sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/communication/Topic.groovy

```diff
@@ -13,6 +13,7 @@ enum Topic {
     NOTIFICATION_SUCCESS("Success notification"),
     NOTIFICATION_INFO("Information notification"),
     SAMPLE_RECEIVED_COUNT_UPDATE("The count of received samples was updated"),
+    SAMPLE_PASSED_QC_COUNT_UPDATE("The count of samples passing QC was updated"),
     SAMPLE_FAILED_QC_COUNT_UPDATE("The count of samples failing QC was updated"),
```

Comment:
You can merge this case with the failure case I think. But you do not have to, we can also merge it in the view only atm.

Reply:
I would leave it for now. We already have an issue for simplifying the service:)